config/environments/production.rb

```diff
@@ -24,8 +24,7 @@
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV["RAILS_SERVE_STATIC_FILES"].present?
 
-  # Compress CSS using a preprocessor.
-  # config.assets.css_compressor = :sass
+  config.assets.css_compressor = nil
```

Comment:
Could you talk about what this is for?

Reply:
Sure. administrate does not work well with tailwind-rails since it still depends on sassc. Without this line, sassc will try to compress and minify all CSS code, including the one from tailwind, which uses features not recognized by sassc.


There are some [ongoing efforts](https://github.com/thoughtbot/administrate/issues/2311) to fix this, but the quickest way at the moment is disabling css compression